potpie/context-engine/adapters/inbound/cli/ui/static_logo_loader.py

```
@@ -10,10 +10,8 @@
 from rich.text import Text
 
 VIEWPORT_WIDTH = 96
-VIEWPORT_HEIGHT = 20
-LOGO_COLOR = "#B6E343"
-LOGO_STYLE = f"bold {LOGO_COLOR} on #000000"
-BLANK_STYLE = "on #000000"
+VIEWPORT_HEIGHT = 12
+from adapters.inbound.cli.ui.brand import LOGO_COLOR, LOGO_STYLE
```

Ruff (0.15.15)

[error] Module level import not at top of file

(E402)